src/main/java/com/ceritoxi/euler4/Euler4.java

```diff
@@ -0,0 +1,20 @@
+package com.ceritoxi.euler4;
+
+import com.ceritoxi.util.Math;
+
+public class Euler4 {
+	
+	private int result;
+	
+	public Euler4() {
+		this.result = Math.largestPalindromeMadeFromTwoXDigitNumberThatAreLessThenSix(3);
```

Comment:
Hmm here i might have some reservations that this particular function in the Math class will never be used in this form ever again.

We should only extract behavior into static classed that might be used a lot of times later.
So isEven or isPalindrom, isPrime all make sense. But this one is too specific.

Lets think about together what part of it can be extracted (if any) and what part should be written here in Euler4 since it will unlikely to be used again.

Reply:
Well this method in particular could be just straight up copied to the Euler4 class, having the argument as a constant.

Reply:
Well for the being, I don't think I will make this change. Though after both this and the 'math' is merged to master I will sure do.